notary-api/src/routes/buyers.js

```diff
@@ -212,23 +210,28 @@ router.post(
  * @swagger
  * /notarization-request:
  *   post:
+ *     description: Receives a batch of sellers to notarize
  *     parameters:
- *       - name: orderId
- *         description: The unique identifier for the order.
- *         required: true
- *         type: number
- *         in: body
- *       - name: sellers
- *         description: List of sellers send to notarize.
+ *       - in: body
+ *         name: notarizationRequest
  *         required: true
- *         type: array
- *         in: body
- *       - name: callbackUrl
- *         description: Buyer's URL to call after notarization.
- *         required: true
- *         type: string
- *         in: body
- *     description: Receives a batch of sellers to notarize
+ *         schema:
+ *           required:
+ *             - orderId
+ *             - sellers
+ *             - callbackUrl
+ *           properties:
+ *             orderId:
+ *               type: integer
+ *               description: The unique identifier for the order
+ *             sellers:
+ *               type: array
+ *               description: List of sellers send to notarize
+ *               items:
+ *                 type: string
```

Comment:
This is also expected as an object
```suggestion
 *                 type: object
```